### PR TITLE
Workout name polish

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by HermenegildoMI — APPROVED** [review]

File: javascript/javascript-playground-questions/js-miles-to-kilometres/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-name: Miles To Kilometres
+name: Miles to Kilometres
 
-description: Create a program to transform Miles to Kilometres or vice versa.
+description: Create a program to transform miles to kilometres or vice versa.
```
Comment (nem035) [inline javascript/javascript-playground-questions/js-miles-to-kilometres/README.md:3]:
```suggestion
description: Create a program to transform miles into kilometres or vice versa.
```

**Pull request merged**